src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -110,9 +117,23 @@ private void setAccelerator(MenuItem menuItem, KeyCombination keyCombination) {
      * Fills up all the placeholders of this window.
      */
     void fillInnerParts() {
+        ColumnConstraints column1 = new ColumnConstraints();
```

Comment:
I think it would be good to have the columns named after its respective functions, what do you think?